node/metered-channel/src/bounded.rs

```diff
@@ -76,6 +73,23 @@ impl<T> Stream for MeteredReceiver<T> {
 }
 
 impl<T> MeteredReceiver<T> {
+	fn maybe_meter_tof(&mut self, maybe_value: Option<MaybeTimeOfFlight<T>>) -> Option<T> {
+		self.meter.note_received();
+		maybe_value.map(|value| {
+			match value {
+				MaybeTimeOfFlight::<T>::WithTimeOfFlight(value, tof_start) => {
+					// do not use `.elapsed()` of `std::time`, it may panic
+					// `coarsetime` does a saturating sub for all `CoarseInstant` substractions
```

Comment:
Not on Linux/Osx/BSD and non Unix where it performs just a normal subtraction as monotonic timers are guaranteed to be monotonic. But it is just a small detail of course, since it still does not panic realistically.

Reply:
https://doc.rust-lang.org/src/std/time.rs.html#247-261